vueui/src/components/header/default.vue

```diff
@@ -28,20 +64,24 @@ export default {
 .uiTitleWrapper {
   margin: -8px -8px 10px;
   position: relative;
-  padding-bottom: 8px;
   float: none;
   background-color: #b9b9b9;
   clear: both;
+  user-select: none;
+  -ms-user-select: none;
```

Comment:
Do we not have/use autoprefixer? We should probably fix that.

Reply:
Probably.